homeassistant/components/tradfri/__init__.py

```diff
@@ -95,7 +88,7 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
         hass.config_entries.flow.async_init(
             DOMAIN,
             context={"source": config_entries.SOURCE_IMPORT},
```

Comment:
Do we need to remove this too? I see there is mention of the word `import` here.

Reply:
I believe the context can be removed, as there are no source to import from.

Reply:
OK, I've now removed it